tests/unit/test_io_convert.py

```diff
@@ -163,6 +171,20 @@ def test_export_roundtrip(self):
                     container=container,
                     write_path=write_path,
                     export_path=export_path)
+                if self.REFERENCES:
+                    if self.TARGET_FORMAT == "H5":
+                        num_bazs = 10
+                        for i in range(num_bazs):
+                            baz_name = 'baz%d' % i
+                            self.assertIsInstance(exported_container.baz_data.data, ContainerH5ReferenceDataset)
+                            self.assertIs(exported_container.baz_data.data[i], exported_container.bazs[baz_name])
+                    elif self.TARGET_FORMAT == "ZARR":
+                        num_bazs = 10
+                        for i in range(num_bazs):
+                            baz_name = 'baz%d' % i
+                            self.assertIsInstance(exported_container.baz_data.data, ContainerZarrReferenceDataset)
+                            self.assertIs(exported_container.baz_data.data[i], exported_container.bazs[baz_name])
+
```

Comment:
@mavaylon1 according to Codecov, this code seems to not be hit during the tests. It seems strange that the references appear as not tested with "ZARR" as a the target format. 

<img width="1566" alt="Screen Shot 2023-09-29 at 3 43 10 PM" src="https://github.com/hdmf-dev/hdmf-zarr/assets/10999845/59ec61bb-4fcf-4126-91e1-2b1dbd66e624">

Can you please double-check why this is not being used? The CodeCov for this PR is here https://app.codecov.io/gh/hdmf-dev/hdmf-zarr/pull/120